src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Class representing a User
+public class User {
+  private String id;
+  private String email;
+  private String username;
+  private ArrayList<Long> planner;
+  private ArrayList<Long> cookbook;
+  private ArrayList<Long> userRecipes;
+  // shopping list will not be implented until after MVP
+  private ArrayList<String> shoppingList;
+
+  public User(String id, String email, String username) {
+    this.id = id;
+    this.email = email;
+    this.username = username;
+    planner = new ArrayList<Long>();
+    cookbook = new ArrayList<Long>();
+    userRecipes = new ArrayList<Long>();
+  }
+
+  public User(String id, String email, String username, ArrayList<Long> planner, ArrayList<Long> cookbook, ArrayList<Long> userRecipes) {
```

Comment:
When are you going to be using this constructor? Won't a new user always start with empty lists?